job_bundles/houdini_husk_usd_render/README.md

```diff
@@ -0,0 +1,60 @@
+# SideFX Houdini Husk USD Render
+
+## Background
+
+Husk is a CLI application provided with SideFX Houdini that renders Universal Scene Description(USD) files.
+By default Husk renders using the Houdini Karma renderer but Husk supports any Hydra-compatible USD render delegate. 
```

Comment:
The job bundles README has an index with short descriptions of jobs, could you add this job to that index? It might fit as another item in the DCC jobs https://github.com/aws-deadline/deadline-cloud-samples/blob/mainline/job_bundles/README.md#digital-content-creation-dcc-render-jobs